portal-web/test/functional/com/liferay/portalweb/paths/portlet/frontenddataset/DataSet.path

```diff
@@ -484,6 +484,26 @@
 	<td>xpath=(//a[contains(@class,'btn')][contains(@href,'new') or contains(@href,'redirect') or contains(@href,'mvc') or contains(@href,'javascript')][.//*[name()='svg'][contains(@class,'icon-plus')]][not(contains(@class,'borderless'))] | //button[not(contains(@class,'borderless'))][not(contains(@class,'btn-unstyled'))][*[contains(@class,'icon-plus')]])[2]</td>
 	<td></td>
 </tr>
+<tr>
+	<td>MESSAGE_FIELD</td>
+	<td>//div[contains(@aria-labelledby, "${key_option}")]//label[text()='Message']/..//input</td>
+	<td></td>
+</tr>
+<tr>
+	<td>COLLAPSE_OPEN</td>
+	<td>//span[contains(.,'${key_fieldName}')]/../*[*[name()='svg'][contains(@class,'lexicon-icon-angle-down')]]</td>
+	<td></td>
+</tr>
```

Comment:
I think there might be an already created path for this, can you check it? If not, you can leave this path here

Reply:
Yes, I created this same path for the CX tests (`ClientExtension#COLLAPSE_ICON_OPEN`), but since CX is not related to these Data Set tests, I created this one for that purpose. Should I use CX paths in these tests?

Reply:
Hmm there are no general paths created? if not, we can use this paths you've created

Reply:
Did you check this?